etc/debian/control

```diff
@@ -0,0 +1,36 @@
+Source: scancode-toolkit
+Section: unknown
+Priority: optional
+Maintainer: root <root@40825ee16090>
+Build-Depends: debhelper (>= 11), dh-python, python, python-setuptools
+Standards-Version: 4.1.3
+Homepage: <insert the upstream URL, if relevant>
+X-Python-Version: >= 2.6
```

Comment:
For now this has to be 2.7 exactly until the port for Python 3 is complete

Reply:
Ok will take down this one in next iteration.